boards/arm/stm32f3_disco/doc/stm32f3_disco.rst

```diff
@@ -119,6 +121,8 @@ Default Zephyr Peripheral Mapping:
 - LD8 : PE14
 - LD9 : PE12
 - LD10 : PE13
+- LSM303DLHC_I2C_SCL : PB6
+- LSM303DLHC_I2C_SDA : PB7
```

Comment:
Maybe we should wait for this sensor to be supported before documenting its pinmapping on the board?

Reply:
removed